.github/workflows/test_suite.yml

```diff
@@ -0,0 +1,53 @@
+name: Test Suite
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - uses: pre-commit/action@v2.0.0
```

Comment:
Not sure if the [way I had it before](https://github.com/lyft/cartography/blob/c153bad7343320fcfb6a0d3233bf0b00d92b1482/.github/workflows/pull_request.yml#L13-L22) is faster (use cache, pip install, run `pre-commit --run --all-files`). I don't have a strong opinion either way. What do you think?